lib/airbrake/sidekiq.rb

```diff
@@ -1,9 +1,8 @@
+gem 'sidekiq', '~> 3.0'
 require 'sidekiq'
 
-if ::Sidekiq::VERSION >= '3'
-  ::Sidekiq.configure_server do |config|
-    config.error_handlers << lambda do |exception, context|
-      Airbrake.notify_or_ignore(exception, :parameters => context)
-    end
+Sidekiq.configure_server do |config|
+  config.error_handlers << ->(exception, context)
```

Comment:
@seuros did you missed `do` at the end of the line?
`  config.error_handlers << ->(exception, context) do`
Without it I get syntax error.